File: src/app/gkill/api/req_res/update_re_kyou_request.go
package req_res

import "github.com/mt3hr/gkill/src/app/gkill/dao/reps"

type UpdateReKyouRequest struct {
	SessionID string `json:"session_id"`

	ReKyou *reps.ReKyou `json:"rekyou"`

	TXID *string `json:"tx_id"`

	LocaleName string `json:"locale_name"`

	UpdatedKyou *reps.Kyou `json:"updated_kyou"`

	WantResponseKyou bool `json:"want_response_kyou"`
}

File: src/client/classes/api/req_res/update-re-kyou-request.ts
'use strict'

import { ReKyou } from '@/classes/datas/re-kyou'
import { GkillAPIRequest } from '../gkill-api-request'
import { Kyou } from '@/classes/datas/kyou'

export class UpdateReKyouRequest extends GkillAPIRequest {

    rekyou: ReKyou

    tx_id: string | null = null

    want_response_kyou: boolean

    updated_kyou: Kyou | null | null = null

    constructor() {
        super()
        this.rekyou = new ReKyou()
        this.want_response_kyou = false
    }

}


